lib/pymedphys/_experimental/streamlit/apps/transfer_check.py

```diff
@@ -70,199 +61,500 @@ def main():
+
+
+def calc_homogeneity_index(dvh_calcs, target, rx_dose):
+    max_target_dose = dvh_calcs[target].max
+    homogeneity_index = max_target_dose / rx_dose
+    return homogeneity_index
+
+
+def calc_dose_homogeneity_index(dvh_calcs, target):
+    D_95 = dvh_calcs[target].dose_constraint(95).value
+    D_5 = dvh_calcs[target].dose_constraint(5).value
+    DHI = D_95 / D_5
+    return DHI
+
+
+def perform_target_evaluation(dd_input, dvh_calcs):
+    rois = list(dvh_calcs.keys())
+    target = st.selectbox("Select the target structure: ", rois)
+    rx_dose = st.number_input("Input Rx dose in Gy: ")
+
+    min_dose = dvh_calcs[target].min
+    max_dose = dvh_calcs[target].max
+    mean_dose = dvh_calcs[target].mean
+    D_1 = dvh_calcs[target].dose_constraint(1).value
+    D_2 = dvh_calcs[target].dose_constraint(2).value
+    D_98 = dvh_calcs[target].dose_constraint(98).value
+    D_99 = dvh_calcs[target].dose_constraint(99).value
+    conformity_index = calc_conformity_index(dd_input, dvh_calcs, target, rx_dose)
+    homogeneity_index = calc_homogeneity_index(dvh_calcs, target, rx_dose)
+    dhi = calc_dose_homogeneity_index(dvh_calcs, target)
+
+    data = {
+        "Min [Gy]": min_dose,
+        "Max [Gy]": max_dose,
+        "Mean [Gy]": mean_dose,
+        "D1 [Gy]": D_1,
+        "D2 [Gy]": D_2,
+        "D98 [Gy]": D_98,
+        "D99 [Gy]": D_99,
+        "CI": conformity_index,
+        "HI": homogeneity_index,
+        "DHI": dhi,
+    }
+
+    target_df = pd.DataFrame.from_dict(
+        data, orient="Index", columns=[target]
+    ).style.set_precision(2)
+    return st.write(target_df)
+
+
+def compare_to_historical_scores(constraints_df, institutional_history):
+    df = constraints_df.copy()
+    df["Institutional Average"] = "-"
+    for index in df.index:
+        row_key = df["Structure_Key"][index]
+        row_type = df["Type"][index]
+        row_dose = df["Dose [Gy]"][index]
+        row_volume = df["Volume [%]"][index]
+        df["Institutional Average"][index] = float(
+            institutional_history[
+                (institutional_history["Structure_Key"] == row_key)
+                & (institutional_history["Type"] == row_type)
+                & (institutional_history["Dose [Gy]"] == row_dose)
+                & (institutional_history["Volume [%]"] == row_volume)
+            ]["Score"].mean()
+        )
+
+    return df
+
+
+def show_dicom(dicom_table):
+    show_table = st.checkbox("View complete DICOM table.")
+    if show_table:
+        st.subheader("DICOM Table")
+        st.dataframe(dicom_table, height=1000)
+
+
+def show_mosaiq(mosaiq_table):
+    show_table = st.checkbox("View complete Mosaiq table.")
+    if show_table:
+        st.subheader("Mosaiq Table")
+        st.dataframe(mosaiq_table, height=1000)
+
+
+def add_alias(dvh_calcs, ALIASES):
+    define_alias = st.checkbox("Define a new structure alias")
+    if define_alias:
+        add_new_structure_alias(dvh_calcs, ALIASES)
```

Comment:
All caps names are usually used to denote "constants", so I don't think that fits here.